ethereum/blockcreation/src/main/java/tech/pegasys/pantheon/ethereum/blockcreation/BlockMiner.java

```diff
@@ -80,6 +80,11 @@ public void run() {
         Thread.currentThread().interrupt();
       } catch (final Exception ex) {
         LOG.error("Block mining threw an unhandled exception.", ex);
+        try {
```

Comment:
Did you run into an issue with the miner when running this?

Reply:
I have a feeling this is a stray change that somehow got in here.  I don't think I've tested the miner at all with this.  There's no reason it should be affected (but software is full of surprises...)